app/facades/foods_facade.rb

```diff
@@ -0,0 +1,14 @@
+class FoodsFacade
+
+  def food_search_results(query)
+    json = FoodsService.new.search_foods(query)
+    json[:foods].map do |food_data|
+      Food.new(food_data)
+    end.first(10)
+  end
+
+  def searched_food_count(query)
+    json = FoodsService.new.search_foods(query)
```

Comment:
I like that you were separating concerns with these two methods, but you're still calling the API twice - which means you have the opportunity here to improve efficiency by 100% 😉 

Can you think of a way to get the `totalHits` data without calling the API again?